gcc4mbed/external/mbed/libraries/mbed/targets/hal/TARGET_NXP/TARGET_LPC43XX/PeripheralNames.h

```diff
@@ -45,7 +48,23 @@ typedef enum {
     ADC1_4,
     ADC1_5,
     ADC1_6,
-    ADC1_7
+    ADC1_7,
```

Comment:
These changes should be part of the mbed SDK. I will prepare a Push Request to mbed.